AEPAssurance.xcodeproj/project.pbxproj

```diff
@@ -944,15 +944,15 @@
 			knownRegions = (
 				en,
```

Comment:
Is there a en-US as well?

Reply:
Good call. Apple defaults to use en, but i will include an en-US just in case.